native_client/deepspeech.cc

```diff
@@ -342,6 +343,34 @@ DS_EnableExternalScorer(ModelState* aCtx,
   return DS_ERR_OK;
 }
 
+
+int
+DS_EnableHotWords(ModelState* aCtx,
+                  const char* aHotWords)
+{
+
+  std::string hot_words_(aHotWords);
+  std::size_t last = 0;
+  std::size_t next = 0;
+  std::string delim = ",";
```

Comment:
woo, so you pass a "list of string" as a `char*` but in fact it is delimited by commas?